guides/assets/javascripts/guides.js

```diff
@@ -23,10 +23,7 @@
     for(var i = 0; i < array.length; i++) callback(array[i]);
   }
 
-  // Viewable on local
-  if (window.location.protocol === "file:") Turbolinks.supported = false;
```

Comment:
Hi Marco. Can you still view the guides locally without this line?

Reply:
Hey @p8, yes that's still possible. It doesn't use Turbo, but it falls back to using `window.location.href = '...'` if it's not "visitable" which is the case for URLs starting with `file://`. See: https://github.com/hotwired/turbo/blob/aeeaae8edb5f6ec6ef6b19eaf93dc060a1e67b10/src/core/drive/navigator.ts#L28-L31

Reply:
That makes sense. Thanks!

Reply:
Works great locally.